src/importer.cpp

```diff
@@ -777,6 +778,7 @@ void flattenComponentTree(const ComponentEntityPtr &parent, ComponentPtr &compon
 
 ModelPtr Importer::flattenModel(const ModelPtr &model)
 {
+    pFunc()->removeAllIssues();
```

Comment:
`flattenModel` documentation should be updated to reflect this behaviour.

Reply:
0d138f4